app/routes/cart.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    // get the cart our of local storage (this way it persists in the user's browser past a reload)
+    let cart = JSON.parse(localStorage.getItem("cart"));
```

Comment:
In a real-world application, do you think using local-storage is an appropriate choice?